src/components/Favorites/Favorite.spec.js

```diff
@@ -0,0 +1,3 @@
+it('should 1 be 1', () => {
+  expect(1).toBe(1);
```

Comment:
What is this testing D:

Reply:
It's filler test so that Travis passes 🙃